src/arrays/programs_involving_data_sequences/hard/q8/MainTest.java

```diff
@@ -0,0 +1,96 @@
+package arrays.programs_involving_data_sequences.hard.q8;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.utils.ArrayUtil.*;
+import static org.junit.Assert.assertArrayEquals;
+
+public class MainTest extends BaseTest {
+    // Java
+
+    public static int n = 10;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = arrayToInput(generateRandomArray(-10, 10, n));
+        return new Clause[]{
+                new StringLiteral("Enter an array of " + n + " integers:"),
+                new Optional(new StringLiteral(" ")),
+                new NewLine(),
+                new StringLiteral("The result of multiplied array is:"),
+                new NewLine(),
+                new PlaceHolder(),
+                new Optional(new StringLiteral(" "))
```

Comment:
```suggestion
```

Reply:
The PlaceHolder will capture all the input after the NewLine so this will never capture anything